src/features/courses/course-route.ts

```diff
@@ -941,6 +941,31 @@ router.get('/topics',
         next(httpResponse.Ok('Fetched successfully', result));
     }));
 
+router.post('/:id/email',
+    authenticationMiddleware,
+    validate(emailProfValidation),
+    // This is due to a typescript issue where the type mismatches extractMap
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    asyncHandler(async (req: RederlyExpressRequest<any, unknown, EmailProfRequest.body, EmailProfRequest.query>, res: Response, next: NextFunction) => {
+        const params: EmailProfRequest.params = req.params;
+        const body = req.body as EmailProfRequest.body;
```

Comment:
You don't need this...

Reply:
Look man, I was having some typing problems!